core/src/test/java/org/elasticsearch/index/replication/ESIndexLevelReplicationTestCase.java

```diff
@@ -197,7 +201,7 @@ protected ReplicationGroup createGroup(int replicas) throws IOException {
         return new ReplicationGroup(metaData, homePath);
     }
 
-    private DiscoveryNode getDiscoveryNode(String id) {
+    protected DiscoveryNode getDiscoveryNode(String id) {
         return new DiscoveryNode(id, id, LocalTransportAddress.buildUnique(), Collections.emptyMap(),
```

Comment:
wondering if we should switch `LocalTransportAddress.buildUnique()` to `new LocalTransportAddress(id)`